.ci/openshift_integration.sh

```diff
@@ -31,15 +31,27 @@ export ODO_DISABLE_TELEMETRY=true
 IMG="$(echo $REGISTRY_IMAGE | cut -d':' -f1)"
 TAG="$(echo $REGISTRY_IMAGE | cut -d':' -f2)"
 
+# Allow for yq dependency to be set based off desired architecture
+DEFAULT_ARCH="amd64"
+
+# Check if different architecture was passed for image build
+# Will default to $DEFAULT_ARCH if unset
+if [ ! -z "$1" ]
+  then
+    arch="$1"
+else
+    arch="$DEFAULT_ARCH"
+fi
```

Comment:
This might fix the error being thrown:
```suggestion
if [ $# -gt 0 ]
  then
    arch="$1"
else
    arch="$DEFAULT_ARCH"
fi
```